scipy/integrate/_ivp/rk.py

```diff
@@ -382,6 +382,173 @@ class RK45(RungeKutta):
         [0, 40617522/29380423, -110615467/29380423, 69997945/29380423]])
 
 
+class Verner87(RungeKutta):
+    """A 'most efficient' Runge--Kutta (13:8(7)) pair.
+
+    References
+    ----------
+    .. [1] J. H. Verner, "Explicit Runge-Kutta methods with estimates
+           of the local truncation error", SIAM Journal on Numerical Analysis,
+           Vol. 15, No. 4, pp. 772-790, 1978.
+    """
+
+    order = 8
+    error_estimator_order = 7
+    n_stages = 13
```

Comment:
I essentially could not discard the last 3 stages as I said in https://github.com/scipy/scipy/pull/14956#issuecomment-956497109 because the sum of weights was not 1.0 and the numerical results were totally off. Therefore, I included again all 13 stages for A, expanded B and Bh too.